manifests/grafana/import-dashboards/job.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         - -c
         - >
           set -x;
-          while [ $(curl -sw '%{http_code}' "http://grafana:3000" -o /dev/null) -ne 200]; do
+          while [ $(curl -sw '%{http_code}' "http://grafana:3000" -o /dev/null) -ne 302 ]; do
```

Comment:
It's better to use `while [ $(curl -L -sw '%{http_code}' "http://grafana:3000" -o /dev/null) -ne 200 ]; do` - to add key *-L* to follow redirection. That's more safe.